pytket/docs/index.rst

```diff
@@ -81,14 +81,14 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     getting_started.rst
     changelog.rst
     install.rst
-    faqs.rst 
+    faqs.rst
 
 .. toctree::
     :caption: More Documentation:
     :maxdepth: 1
     
     Manual <https://cqcl.github.io/pytket/manual/index.html>
-    Extensions <https://cqcl.github.io/pytket-extensions/api/index.html>
+    extensions_index.rst
```

Comment:
Have you checked if this is showing up as intended? Do we want to update the links back from the extensions to this overview as well?

Reply:
Here's what it looks like before and after I click. I would like to avoid the dropdown as this looks really bad.

Any ideas how to fix this?

I've tried the following
```
.. toctree::
    :caption: More Documentation:
    :maxdepth: 0
```
But that doesn't work

![Screenshot 2023-10-31 at 18 50 33](https://github.com/CQCL/tket/assets/93673602/019daee8-3ac0-42fd-8e1e-c9e6ca86a858)

![Screenshot 2023-10-31 at 18 50 23](https://github.com/CQCL/tket/assets/93673602/f361aec6-cb5c-4912-ac3e-8f3445f04a3d)

Reply:
The dropdown looks not like the best solution, but showing them always is taking a lot of space also... Do you have any other ideas how to handle this?